### ISSUE TITLE
IPV6 Support

### ISSUE THREAD

**Issue by ghpsk** [overte-org/overte#713]
Google DNS is hardcoded in, and when that fails a hardcoded IPv4 literal is used (192.168.0.1).
IPv4 literals are flooding the log. They're not used everywhere, but their usage is substantial enough to be concerning.

IPv4 literals will cause issues on IPv6 networks unless something is done about them.
Automatically using available transition technologies such as NAT64 and DNS64 on IPv4 addresses would be ideal.
Additionally, the software should detect network configuration from the operating system rather than making a guess. In Linux, DNS addresses can be obtained via the /etc/resolv.conf file.

Logins are possible on an IPv6-only network, but joining worlds is not.

**Comment by daleglass** [comment]
Yes, we definitely want IPv6 to be supported properly eventually.

Where is 192.168.0.1 hardcoded? I searched the code and didn't find it. Can you provide sample logs please?

**Comment by ksuprynowicz** [comment]
I opened a work in progress PR for IPv6 support. Our codebase is very big though, so help is very welcome - there will be a lot of work.
https://github.com/overte-org/overte/pull/715
